docs/using-with-xcodebuild.md

```diff
@@ -0,0 +1,16 @@
+# Using With xcodebuild
+
+## Custom SourcePackages dir (via `-clonedSourcePackagesDirPath`)
+The plugin should work with both Xcode and xcodebuild.
+
+The plugin uses the `SOURCE_PACKAGES_CHECKOUTS_DIR` setting (resolved from `BUILD_ROOT`) to handle linking and other build configurations. This corresponds to the SourcePackages dir (containing SPM packages' sources). If you use a custom SourcePackages dir in xcodebuild, via the `-clonedSourcePackagesDirPath` argument, the plugin cannot infer this dir correctly. In this case, please pass the `SOURCE_PACKAGES_CHECKOUTS_DIR` setting to the xcodebuild command.
+
+```sh
+xcodebuild archive \
+  -workspace App.workspace \
+  -scheme App \
+  ... \
+  -clonedSourcePackagesDirPath=a/b/c \
+  ...
+  SOURCE_PACKAGES_CHECKOUTS_DIR=a/b/c # <---HERE
+```
```

Comment:
Nit: The `-clonedSourcePackagesDirPath` argument works as a relative path, but near as I can tell, `SOURCE_PACKAGES_CHECKOUTS_DIR` has to be an absolute path (at least, I'm not sure what directory it's resolved relative to otherwise, but I don't think its the CWD).

Reply:
Noted. Actually, paths in build settings should be absolute while paths in CLI arguments can be relative or absolute.
I'll update the doc to make this clear.